examples/basic.ipynb

```diff
@@ -0,0 +1,363 @@
+   "source": [
+    "harmony_client = Client(auth=(username.value, password.value))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "brown-eclipse",
+   "metadata": {},
+   "source": [
+    "Next, we create a Collection object with the CMR collection id for our test collection. We then create a Request which specifies the collection, and a `spatial` `BBox` describing the bounding box for the area we're interested in. We'll see later in the notebook how to make sure the request we have is valid."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "dental-contribution",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "collection = Collection(id='C1234088182-EEDTEST')\n",
+    "\n",
+    "request = Request(\n",
+    "    collection=collection,\n",
+    "    spatial=BBox(-165, 52, -140, 77)\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "accurate-stone",
+   "metadata": {},
+   "source": [
+    "Now that we have a request, we can submit it to Harmony using the Harmony Client object we created earlier. We'll get back a JSON object that describes the Harmony job that we've submitted. As you can see, the Job contains the original request that was submitted to Harmony, its status, as well as a URL where we can get the Job's current status, and its results if it has completed."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "excited-mount",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "job1 = harmony_client.submit(request)\n",
+    "\n",
+    "helper.JSON(job1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "gothic-intent",
+   "metadata": {},
+   "source": [
+    "Now using our helper module, we'll wait for the Job to complete, and then we'll download the results and view them."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "postal-bibliography",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "helper.download_and_show_results(harmony_client, job1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "early-package",
+   "metadata": {},
+   "source": [
+    "Now we show a Harmony request for a temporal range: one month in 2020. As before, we create a Request, and submit it with the same Harmony Client we used above."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "statutory-antigua",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "request = Request(\n",
+    "    collection=collection,\n",
+    "    temporal={\n",
+    "        'start': dt.datetime(2020, 6, 1),\n",
+    "        'stop': dt.datetime(2020, 6, 30)\n",
+    "    })\n",
+    "\n",
+    "job2 = harmony_client.submit(request)\n",
+    "\n",
+    "helper.JSON(job2)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "detected-quarter",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "download_and_show_results(job2)"
```

Comment:
needs module qualifier and extra arg: helper.download_and_show_results(harmony_client, job2)